yt/funcs.py

```diff
@@ -137,7 +137,7 @@ def humanize_time(secs):
     """
     mins, secs = divmod(secs, 60)
     hours, mins = divmod(mins, 60)
-    return "%02d:%02d:%02d" % (hours, mins, secs)
+    return ":".join(f"{t:02}" for t in (hours, mins, secs))
```

Comment:
this is one case where i vastly prefer the older string formatting syntax :) 

but more importantly, if the incoming `secs` is a float (which it probably is? hard to say, this function doesn't seem to be used anywhere in yt...), the formatting differs significantly, should cast to float to keep the behavior identical:

```suggestion
    return ":".join(f"{int(t):02}" for t in (hours, mins, secs))
```

For reference, as written, the new version looks like

```
>>> humanize_time(1000.1)
'0.0:16.0:40.10000000000002'
```
vs the old

```
>>> humanize_time(1000.1)
'00:16:40'
```